.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
```

Comment:
sorry but how does this not apply spaces instead of tabs to all files?

i found this somewhere, seems to be closer, maybe start with this? (unless I have missed the point of this PR)

```
[*]
end_of_line = lf
charset = utf-8
indent_style = tab
indent_size = 4
insert_final_newline = true
trim_trailing_whitespace = true

[*.json]
insert_final_newline = false

[package.json]
indent_style = space
indent_size = 2

[*.md]
indent_style = space
trim_trailing_whitespace = false

[*.yml]
indent_style = space
indent_size = 2
```

Reply:
Yes, I created that file :) That was the original one that I was going to include here, but there are no files in this repo besides markdown files (and the LICENSE file), so I thought that it doesn't make much sense to include config for other files.

That said, maybe I should have included only the config for Markdown files. Or, it it makes it clearer, I can include the whole file.

Reply:
lulz :) One file to rule them all, then link out to it?